politeiawww/cmd/piwww/piwww.go

```diff
@@ -50,8 +50,9 @@ type piwww struct {
 	ProposalSetStatus ProposalSetStatusCmd `command:"proposalsetstatus"`
 
 	// Comments commands
-	CommentNew  CommentNewCmd  `command:"commentnew" description:"(user)   create a new comment"`
-	CommentVote CommentVoteCmd `command:"commentvote" description:"(user)   upvote/downvote a comment"`
+	CommentNew       CommentNewCmd       `command:"commentnew" description:"(user)   create a new comment"`
+	CommentVote      CommentVoteCmd      `command:"commentvote" description:"(user)   upvote/downvote a comment"`
+	ProposalComments ProposalCommentsCmd `command:"proposalcomments" description:"(public) get the comments for a proposal"`
```

Comment:
Lets make this just `Comments`.  It's piwww so you know its proposal comments.  It also makes it easier to find the comment commands when they all start with Comment.